.mage/js_sdk.go

```diff
@@ -149,6 +153,10 @@ func (k JsSDK) Definitions() error {
 
 // DefinitionsClean removes the generated api-definition.json.
 func (k JsSDK) DefinitionsClean(context.Context) error {
+	err := sh.Rm(filepath.Join("sdk", "js", "generated", "allowed-field-mask-paths.json"))
+	if err != nil {
+		return err
```

Comment:
Please make sure that we don't return on a "does not exist" and "forget" to delete api-definition.json

Reply:
Tested now - it doesn't return error on not found and continues even if the file is missing.